pandas/tests/series/test_operators.py

```diff
@@ -1587,7 +1606,25 @@ def test_operators_reverse_object(self, op):
         expected = op(1., arr.astype(float))
         assert_series_equal(result.astype(float), expected)
 
-    def test_operators_combine(self):
+    pairings = []
+    for op in ['add', 'sub', 'mul', 'pow', 'truediv', 'floordiv']:
```

Comment:
I think we already have these no as fixtures?

Reply:
conftest.py doesn't have one specifically for this subset.  Figuring out what is worth fixturizing will be much easier after arithmetic tests are centralized (#22033 and follow-ups)

Reply:
sure we do, all_arithmetic_operators (and several slices of this)

Reply:
See above re subsetting fixtures.